src/components/views/room_settings/AliasSettings.js

```diff
@@ -99,11 +118,6 @@ export default class AliasSettings extends React.Component {
         return dict;
     }
 
-    isAliasValid(alias) {
-        // XXX: FIXME https://github.com/matrix-org/matrix-doc/issues/668
-        return (alias.match(/^#([^/:,]+?):(.+)$/) && encodeURI(alias) === alias);
```

Comment:
we lost this validation somewhere...

Reply:
As per the title of the PR: Delegate all room alias validation to the RoomAliasField validator, i.e: https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/components/views/elements/RoomAliasField.js#L80-L86

Reply:
/me tries to not flip a table at the duplication

Reply:
hence fixing https://github.com/vector-im/riot-web/issues/11737 :P

Reply:
hence fixing vector-im/riot-web#11737 :P